docs/src/openapi-v1.yaml

```diff
@@ -564,6 +564,59 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/RuntimeSpec'
+  /pallets/{palletId}/storage:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of storage items for a pallet.
+      description: Returns a list of storage item metadata for storage items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to query the storage of. Note: pallet name aligns with
+          pallet name as specified in runtime metadata.'
+        required: true
+        schema:
+          type: string
+      - name: onlyIds
+        in: query
+        description: Only return the names (IDs) of the storage items instead of all of each storage
+          items metadata.
+        required: false
+        schema:
+          type: boolean
+      - name: at
+        in: query
+        description: Block at which to retrieve a list of
+          the pallet's storage items.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
```

Comment:
Does this appear as-is in the rendered docs? Not sure if `$hex` is immediately obvious to readers.

Reply:
This is a bit of a hack - when rendered it appears as `$unsignedInteger or $hex`.

Reply:
Oh dear, that's pretty weird?